tools/templates/presentation/templates/themeselector.tpl.html

```diff
@@ -133,260 +156,80 @@ <h4>
-  .font-select .fs-search {
-  border-bottom: 1px solid #aaa;
-  padding: 4px;
-  }
-
-  .font-select .fs-search input {
-  padding: 7px;
-  width: 100%;
-  border: 1px solid #aaa;
-  font: 16px Helvetica, Sans-serif;
-  box-shadow: inset 0 1px 3px rgba(0,0,0,.06);
-  border-radius: .1875rem;
-  }
-
-</style>
-<script>
-  function openNav() {
-    // si c'est déja ouvert, on ferme
-    if (document.getElementById("preset-sidenav").style.width == "250px") {
-      closeNav()
-    } else {
-      document.getElementById("preset-sidenav").style.width = "250px";
-      document.getElementById("yw-container").style.paddingRight = "250px";
-    }
-    return false;
-  }
-
-  function closeNav() {
-    document.getElementById("preset-sidenav").style.width = "0";
-    document.getElementById("yw-container").style.paddingRight = "0";
-    return false;
-  }
-  document.addEventListener('DOMContentLoaded', function(){
-    if (typeof $('.colorpicker').spectrum === "function") {
-      $('.colorpicker').spectrum({
-        showPalette:true,
-        showAlpha: true,
-        showInput: true,
-        clickoutFiresChange: true,
-        showInitial: true,
-        chooseText: "Sauver",
-        cancelText: "Annuler",
-        change: function(color) {
-          document.documentElement.style.setProperty('--'+$(this).attr('name'), color.toRgbString());
-        },
-        hide: function(color) {
-          document.documentElement.style.setProperty('--'+$(this).attr('name'), color.toRgbString());
-        },
-        move: function(color) {
-          document.documentElement.style.setProperty('--'+$(this).attr('name'), color.toRgbString());
-        },
-        palette: [
-          [
-            getComputedStyle(document.documentElement).getPropertyValue('--primary-color'),
-            getComputedStyle(document.documentElement).getPropertyValue('--secondary-color-1'),
-            getComputedStyle(document.documentElement).getPropertyValue('--secondary-color-2')
-          ],
-          [
-            getComputedStyle(document.documentElement).getPropertyValue('--neutral-light-color'),
-            getComputedStyle(document.documentElement).getPropertyValue('--neutral-soft-color'),
-            getComputedStyle(document.documentElement).getPropertyValue('--neutral-color')
-          ]
-        ]
-      })
-    }
-    if (typeof $('.fontpicker').fontselect === "function") {
-      $('.fontpicker')
-      .fontselect({
-        placeholder: 'Choisir une police',
-        placeholderSearch: 'Chercher...',
-      })
-      .on('change', function() {
-        // Replace + signs with spaces for css
-        var font = this.value.replace(/\+/g, ' ');
-
-        // Split font into family and weight
-        font = font.split(':');
-
-        var fontFamily = font[0];
-        var fontWeight = font[1] || 400;
-
-        document.documentElement.style.setProperty('--'+$(this).attr('name'), "'"+fontFamily+"'");
-      });
-    }
-  }, false);
-</script>
+<SCRIPT>
```

Comment:
PAS DE MAJUSCULES POUR LES BALISES HTML!!!! ;)
cf. https://stackoverflow.com/questions/19808514/is-it-bad-to-use-uppercase-letters-for-html-tags

Reply:
oups :blush: